Homework_RalucaHotea/CodeChallenge3/app/src/main/java/com/example/hommework3/MainActivity.java

```diff
@@ -0,0 +1,74 @@
+package com.example.hommework3;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+import android.os.Bundle;
+import android.text.TextUtils;
+import android.util.Patterns;
+import android.view.View;
+import android.widget.Button;
+import android.widget.CheckBox;
+import android.widget.CompoundButton;
+import android.widget.EditText;
+import android.widget.TextView;
+
+public class MainActivity extends AppCompatActivity {
+
+    private EditText emailContent;
+    private EditText phoneContent;
+    private Button submitButton;
+    private TextView textResult;
+    private CheckBox checkBox;
+    private boolean checked;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.my_first_view);
+
+        submitButton = findViewById(R.id.button_id);
+
+        submitButton.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                emailContent = findViewById(R.id.email_id);
+                phoneContent = findViewById(R.id.phone_id);
+
+                boolean isEmailValid = isValidEmail(emailContent.getText().toString());
+                boolean isPhoneValid = isValidPhone(phoneContent.getText().toString());
+
+                if(isEmailValid && isPhoneValid && checked){
```

Comment:
Please consider formatting the code with CTRL  + ALT + L and avoid adding empty lines of code because it increases the size of the class.